README.md

```diff
@@ -71,11 +81,21 @@ Returns a promise that resolves to an array of search results. Returns `null` if
 
 Receives the same arguments as `getLyrics`.
 
-### `getLyricsFromPath(path: string)`
+### `getData(path: string)`
 
 Get the lyrics of a song using the `path` property present in the search results of `searchLyrics` method.<br/>
+Returns a promise that resolves to a string containing html data.
+
+### `getLyricsFromPath(data: string)`
+
+Get the lyrics of a song using the `data` property present in the search results of `getData` method.<br/>
```

Comment:
Since getLyricsFromPath now accepts html data instead of a path, it should be renamed to something like `getLyricsFromData`. I think we should also keep the original `getLyricsFromPath`, in case a user wants to handle the lyrics search/fetch manually.